src/extractLoader.js

```diff
@@ -91,7 +91,8 @@ function evalDependencyGraph({loaderContext, src, filename, publicPath = ""}) {
             exports,
             __webpack_public_path__: publicPath, // eslint-disable-line camelcase
             require: givenRelativePath => {
-                const indexOfQuery = givenRelativePath.includes("?") ? givenRelativePath.indexOf("?") : givenRelativePath.length;
+                const noDoubleQuestionMarkPath = givenRelativePath.replace(/\?\?/g, "##");
```

Comment:
When is `??` part of a valid url? Also if you handled `??`, why didn't you also handle `???`?

I don't think you need to handle these anyways. You should just focus on fixing the query part which was handled wrongly in the original code and leave these other cases to the judgement of the person using it.

Reply:
There was a test case failing, it was related to double question mark. I believe it is valid in webpack context.